### PR TITLE
Add Urdu language

### PR DESCRIPTION
**Pull request by AalianKhan**
Hello, I added most Sentences fr Urdu. But I am no sure how to test the files as of right now.

### CODE REVIEW

**Comment by AalianKhan** [comment]
It's looking for voluptuous module. When I try to run ```pip install voluptuous``` it says file not found

**Comment by synesthesiam** [comment]
@AalianKhan, did you run the `script/setup`?

**Comment by AalianKhan** [comment]
I ran ```script/setup``` on the terminal but didn't see any new venv path

**Comment by synesthesiam** [comment]
It should be created in a `venv` directory inside the repo. So `source venv/bin/activate` would activate it.

**Comment by AalianKhan** [comment]
Don't see it
image
I am running this command
image

**Comment by synesthesiam** [comment]
Are you able to [create a codespace](https://github.com/home-assistant/intents/blob/main/docs/codespace/README.md) instead? This will already have everything set up.

**Comment by AalianKhan** [comment]
Hello,
Urdu uses the same sentence to close a cover and turn off a light. How should I proceed?
image

**Comment by synesthesiam** [comment]
@AalianKhan I'll be working on a solution for the HassTurnOff/HassCloseCover issue today

File: sentences/ur/_common.yaml [hunk 1]
```diff
@@ -1,12 +1,107 @@
...
+        out: "celsius"
+
+expansion_rules:
+  name: "{name} [کو] [کے] [کا]"
+  area: "{area} [کا] [کے] [کو] [میں]"
+  temp: "(temperature | درجہ حرارت | ٹیمپریچر)"
+  temperature: "{temperature} [ڈگری] [degree] [درجہ حرارت] [{temperature_unit}]"
+  brightness: "{brightness:brightness_pct} [percent | پرسنٹ]"
+  set: "[میں] [پر] (کر دیں | سیٹ کردو | کردو | رکھلو | دالدو | دالو | سیٹ کرو | کرو | کر)"
+  all: "(تمام | سب | ساری | سارے)"
```
Comment (balloob) [inline sentences/ur/_common.yaml:102]: Expansion rules that do not wrap a slot should be written in the native language.

Comment (AalianKhan) [reply sentences/ur/_common.yaml:102]: I am a bit confused about what you mean by "wrap a slot". Do you want me to write set in urdu?

Comment (AalianKhan) [reply sentences/ur/_common.yaml:102]: BTW I would like to be a language leader as well

**Comment by yousaf465** [comment]
@AalianKhan Cover open can be 
"پردہ کھولنا" 

Light turn off can be                                                                                                                                                             "بتی بند کریں"

**Comment by AalianKhan** [comment]
@yousaf465 Thanks a lot. I will have these added.
On a different note, I want your opinion on something. You know how in Urdu we often write English words with the Urdu alphabet, do you think we should include these variations or insert the English word with the English alphabet?
such as براؤن is the same thing as brown but just in the Urdu alphabet.

**Comment by yousaf465** [comment]
> @yousaf465 Thanks a lot. I will have these added. On a different note, I want your opinion on something. You know how in Urdu we often write English words with the Urdu alphabet, do you think we should include these variations or insert the English word with the English alphabet? such as براؤن is the same thing as brown but just in the Urdu alphabet.

I think we should go with the Urdu alphabet avoiding Roman urdu, instead stick to Nastaʿlīq script.  https://en.wikipedia.org/wiki/Roman_Urdu

**Comment by yousaf465** [comment]
> @yousaf465 forgot to include this in my last reply. The reason of putting Roman Urdu is because the goal of this project is to collect all possible intents that a person can say whether that's via text or speech. Roman Urdu is a popular way of communicating, but it isn't traditional at all.
> 
> Do you have any thoughts @balloob

Yes, we can use it in aliases, but since the spelling are not standardised we might have issues there.

**Pull request merged**